firmware/quadruna/BMS/src/io/ltc6813/io_ltc6813CellTemps.c

```diff
@@ -1,327 +0,0 @@
-#include <string.h>
```

Comment:
If any amount of this file was used as the basis one the new file, use `git mv` to maintain file history